src/ResourceManager/ContainerInstance/Commands.ContainerInstance/Commands/NewAzureContainerGroupCommand.cs

```diff
@@ -46,6 +47,12 @@ public class NewAzureContainerGroupCommand : ContainerInstanceCmdletBase
             ParameterSetName = CreateContainerGroupWithRegistryParamSet,
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "The resource group name.")]
+        [Parameter(
```

Comment:
If a parameter is in all sets, and Mandatory and in the same position, you don't need to have a separate attribute for each parameter set.  Please erase all parameter attributes and put in:
```
[Parameter(
Mandatory = true,
Position = 0,
ValueFromPipelineByPropertyName = true,
HelpMessage = "The resource group name.")]
```
Apply this all other parameters also.  Especially if you plan to add more parameter sets in the future, there is no need to create a new attribute each time.